src/components/Input/Input.tsx

```diff
@@ -10,6 +10,10 @@ import styles from './Input.module.scss';
 
 export interface InputProps extends Omit<AriaTextFieldProps, 'value' | 'onChange'> {
   className?: string;
+  wrapperClassName?: string;
+  inputClassName?: string;
+  alertClassName?: string;
+  helperTextClassName?: string;
```

Comment:
nit: I generally don't like exposing all the internal class names as it will most likely lead to components being restyled (and thus breaking the design system). It looks like `SelectInput` does a relatively small amount of restyling on the base `Input` component - perhaps we could do this a different way which avoids exposing extra class props.

Reply:
Yeah, I agree. Extra className props in `Input` are a bit messy :(

Is this okay as a temp solution? Once the `DropdownMenu` 2.0 changes go through we can delete this `SelectInput` and remove the additional classNames from `Input`.

Reply:
Yep we'll assess it as an iteration